ui/raidboss/data/06-ew/hunts/thavnair.ts

```diff
@@ -145,6 +145,13 @@ const triggerSet: TriggerSet<Data> = {
       condition: (data) => data.inCombat,
       response: Responses.getBehind(),
     },
+    {
+      id: 'Hunt Yilan Mini Light',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A5F', source: 'Yilan', capture: false }),
+      condition: (data) => data.inCombat,
+      response: Responses.getOut(),
```

Comment:
Do you think it's worth only doing this if you don't have a march debuff?

Reply:
No, I think it's better to call always